extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -146,6 +146,9 @@ public function getWikiVariables() {
 			'theme' => SassUtil::normalizeThemeColors( SassUtil::getOasisSettings() ),
 			'tracking' => [
 				'vertical' => HubService::getVerticalNameForComscore( $wgCityId ),
+				'comscore' => [
+					'c7Value' => AnalyticsProviderComscore::getC7Value(),
```

Comment:
It looks that in fact `false` is not possible.

`WikiFactoryHub::getWikiVertical()` always returns vertical name. I think we can remove this `false'y` handling.